src/main/java/com/jalasoft/selenium/daniel/jauregui/movies/Customer.java

```diff
@@ -43,35 +46,70 @@ public String getName() {
 
     /**
      * Generate the Statement Rentals of Customer.
-     * @return : Will return the summary of Customer Statement
      */
-    public String statement() {
-        double totalAmount = 0;
+    public void statement() {
+        for (Rental rental : this.rentals) {
+            appedStatement(rental);
+        }
+    }
+
+    /**
+     * appedStatement.
+     * @param each movie of customer.
+     */
+    public void appedStatement(final Rental each) {
+        this.statement =  new StringBuilder().append(this.statement)
+                .append("\t").append(each.getMovie().getTitle())
+                .append("\t").append(String.valueOf(calculateAmount(each)))
+                .append("\n").toString();
+    }
+
+    /**
+     * calculateAmount.
+     * @param each movie of customer.
+     * @return calculate amount for current movie.
+     */
+    public double calculateAmount(final Rental each) {
+        return each.getMovie().calculateRate(each.getDaysRented());
+    }
+
+    /**
+     * printStatement.
+     * @return the text of Statement
+     */
+    public String printStatement() {
+        return new StringBuilder().append("Rental Record for ")
+                .append(getName()).append("\n")
+                .append(this.statement)
+                .append("Amount owed is ")
+                .append(String.valueOf(calculateTotalAmount()))
```

Comment:
It is not necessary to use the String.valueOf in the StringBuilder. 
Please remove it.